Formula/p/php.rb

```diff
@@ -100,6 +100,9 @@ def install
     # Prevent homebrew from hardcoding path to sed shim in phpize script
     ENV["lt_cv_path_SED"] = "sed"
 
+    # Identify as 'Build by: Homebrew' in php -v output and phpinfo()
+    ENV["PHP_BUILD_PROVIDER"] = "Homebrew"
```

Comment:
This seems nicer, that way if someone copies a Formula it'll not claim to be build by Homebrew.
```suggestion
    ENV["PHP_BUILD_PROVIDER"] = tap.user
```

Reply:
Implemented in all 4 formulas and reworded the comment accordingly